MmSupervisorPkg/Drivers/MmPeiLaunchers/MmIplPei.inf

```diff
@@ -18,11 +18,9 @@
   ENTRY_POINT                    = MmIplPeiEntry
 
 # Secure:
```

Comment:
Sorry, I pushed the wrong change. GIve me a few mins to update the override tags.

Reply:
Updated